### DISCUSSION TITLE
Cairo, pdf and printer dialog support

### DISCUSSION THREAD

**Discussion by Pazystamas** [frang75/nappgui_src#159] [Q&A]
Hello,
I have few questions before starting.
1. Linux uses Cairo, but not Windows. Can I use directly Cairo on Linux and Windows for drawing and then to save pdf and svg? Or there is a reason to not use it on windows?
2.  Is there a simple solution for showing Printer dialog on all supported OS and then print view or saved pdf?
3. Did I miss or there is no themes for GUI?
Thank you!

**Comment by frang75** [comment]
Hi @Pazystamas 
1. Cairo is an open source cross-platform library for 2D Graphics. Off course, you can use it on Windows. NAppGUI uses Cairo to implement drawing support in Linux, but doesn't offer direct access to Cairo.
2. No, there is not a simple solution. NAppGUI has in its roadmap implement cross-platform printing support but, at the moment, is not prioritary.
3. NAppGUI is themeless. It uses native OS widgets to respect the look and feel of each platform.
